superset-frontend/packages/superset-ui-chart-controls/src/shared-controls/customControls.tsx

```diff
@@ -59,26 +58,12 @@ export const xAxisSortControl = {
   name: 'x_axis_sort',
   config: {
     type: 'XAxisSortControl',
-    label: t('X-Axis Sort By'),
-    description: t('Whether to sort descending or ascending on the X-Axis.'),
-    shouldMapStateToProps: (
```

Comment:
removed due to this hook being flaky (has been disabled on many other similar controls, too)